src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -443,7 +443,7 @@ def get_quotas(self, fs_id, path):
         cfs = self._cephfs_instance(fs_id)
         return cfs.get_quotas(path)
 
-    @RESTController.Resource('POST')
+    @RESTController.Resource('PUT')
     def set_quotas(self, fs_id, path, max_bytes=None, max_files=None):
```

Comment:
```suggestion
    @RESTController.Resource('PUT')
    def quota(self, fs_id, path, max_bytes=None, max_files=None):
```
`quota` instead of `quotas` because we are using the singular form for everything (RESTful practices recommends plural, but that'd imply changing all endpoints).

BTW, is this a partial update?

Reply:
Had a hard time doing making this change with quotas.
Seems like it is working for tree, snapshot, etc but I haven't tested it thoroughly yet